src/node_internals.h

```diff
@@ -258,7 +258,14 @@ class DebugSealHandleScope {
 
 class ThreadPoolWork {
  public:
-  explicit inline ThreadPoolWork(Environment* env) : env_(env) {
+  enum class ThreadPoolWorkType {
+    BLOB,
+    CRYPTO,
+    ZLIB,
+    NAPI,
```

Comment:
n-api has been renamed to node-api. Its type name should be either `NODE_API` or `ADDON`.

```suggestion
    NODE_API,
```

Reply:
Fixed.